pkg/app/appdisc/factory.go

```diff
@@ -81,8 +74,7 @@ func (f *Factory) AppUpdater(conf appcommon.ProcConfig) (Updater, bool) {
 	switch conf.AppName {
 	case skyenv.VPNServerName:
```

Comment:
Should we add the proxy-server back since we are getting rid of heartbeat @jdknives

Reply:
@mrpalide @ersonp yes and yes.

Reply:
Done.